test/integration/testsuite_test.go

```diff
@@ -23,21 +23,23 @@ var userHome string
 var versionInfo VersionAnswer
 
 var bundlePath string
+var ginkgoOpts string
 var pullSecretPath string
 
 func TestTest(t *testing.T) {
 
 	RegisterFailHandler(Fail)
 
 	// fetch the current (reporter) config
-	_, reporterConfig := GinkgoConfiguration()
+	suiteConfig, reporterConfig := GinkgoConfiguration()
+
 	err := os.MkdirAll("out", 0775)
 	if err != nil {
 		logrus.Infof("failed to create directory: %v", err)
 	}
 	reporterConfig.JUnitReport = filepath.Join("out", "integration.xml")
 
-	RunSpecs(t, "Test Suite", reporterConfig)
+	RunSpecs(t, "Test Suite", suiteConfig, reporterConfig)
```

Comment:
Is this change needed to use the labels? maybe add a comment or explain in the commit log why this is needed

Reply:
`suiteConfig.LabelFilter` is a suite-wide variable and it might be useful (e.g. when I thought I'd use it to enforce the use of PULL_SECRET_PATH in case filter is either empty or contains `openshift-preset`. I did not end up doing it, but I think having `suiteConfig` out in the open is better than hiding it behind `_` as I had done before. So I kept it there. Can it stay or should it be removed? It's not needed per se.

Reply:
thanks for explaining,  its just that its not clear how this change relates to the overall "adding of podman preset test" change. so I was trying to guess! :)